src/_variables.scss

```diff
@@ -1,12 +1,22 @@
-$lightest-blue: #e7f0ff;
-$lighter-blue: #d3e3fc;
-$light-blue: #b1cefd;
-$blue: #77a6f7;
+$blue-lightest: #e7f0ff;
+$blue-lighter: #d3e3fc;
+$blue-light: #b1cefd;
+$blue-themed: #77a6f7;
 $teal: #00887a;
-$baby-blue: #00bad3;
-$light-red: #fd8383;
-$lighter-red: #ffccbc;
-$shadow-grey: #777;
+$teal-dark: #009483;
+$blue-baby: #00a5bb;
+$red-light: #fd8383;
+$red-lighter: #ffccbc;
```

Comment:
These shades should probably go into a map like

```scss
$red: (
  "normal": #ff0000,
  "lighter": #ffccbc;
  "light": #fd8383
)
```

Reply:
Done - the only problem is `$teal`, I'm assuming as it is a standard color name it's not possible to have a `$teal` map. I don't understand why it works for `$red` though.